install/kubernetes/cilium/values.yaml

```
@@ -1006,6 +1006,11 @@ hubble:
       runAsGroup: 1001
       fsGroup: 1001
 
+    # -- hubble-ui service configuration.
+    service:
+      type: ClusterIP
+      # nodePort: 31234
```

We can also uncomment `nodePort` with the suggested change above as it is then only used if the service is set to `NodePort`.

```suggestion
      # --- The port to use when the service type is set to NodePort.
      nodePort: 31234
```